### ISSUE TITLE
Preset folder index issue

### ISSUE THREAD

**Issue by taooceros** [Flow-Launcher/Flow.Launcher#137]
**Describe the bug/issue**
In my query result, folders and pdf files's image appears like this. I don't think the image like an error is by design for folder🤨.

**Screenshots**
![image](https://user-images.githubusercontent.com/45326534/90518616-9b4d4600-e199-11ea-99a2-8349543e8b41.png)

**Your System**
```
Windows build number: 10.0.19041.450
Flow Launcher version: 1.20
```
**Flow Launcher Error Log**

[2020-08-18.txt](https://github.com/Flow-Launcher/Flow.Launcher/files/5090649/2020-08-18.txt)

**Comment by jjw24** [comment]
That's the error image. Flow failed to load the image thumbnail for those files and folders. Can you let me know:
- If this is happening to all your folders and files in other locations or just the ones in that location

- Do the images for programs show up?

**Comment by taooceros** [comment]
> That's the error image. Flow failed to load the image thumbnail for those files and folders. Can you let me know:
> 
> * If this is happening to all your folders and files in other locations or just the ones in that location
> * Do the images for other results show up, eg Programs?

I think the image doesn't happen for Program, except for part of the UWP program, but for most of folder and some of the files I think.
![image](https://user-images.githubusercontent.com/45326534/90590490-46e6ac80-e213-11ea-826c-aecf039d421c.png)

For those folder and files without image, I enter the selection and It results only the path, and nothing else like this.
![image](https://user-images.githubusercontent.com/45326534/90590576-831a0d00-e213-11ea-9336-c56311f29d69.png)

But for those UWP program loading image fail, I can open them with Flow.

I think it may be an issue of the indexing of explorer, since it changes name for some of the folder like "document" → "文档", which results these problems.
![image](https://user-images.githubusercontent.com/45326534/90590641-aa70da00-e213-11ea-9a23-89e9e3b994d0.png)

**Comment by jjw24** [comment]
Can you copy the file Program.cs in your screenshot to a new location without the Chinese name eg. D:\Test\
I want to see if it is because of the folder name is in Chinese.

**Comment by taooceros** [comment]
I think it is not due to Chinese issues, but due to the translation to special folder from Windows. Since the language in my windows is Chinese, it will translate all the special folder name to the **preset name** in the index, which is not the original path.

For example, it will translate the folder "document" to 文档, since it means document in Chinese, and "download" to "下载", etc. 

I have tried other folders that has windows index with Chinese path, and it seems they works fine.
![image](https://user-images.githubusercontent.com/45326534/90604988-4a3c6100-e230-11ea-9de6-7145b37fba86.png)

I am sorry that it may not be possible for me to test it with windows sandbox since the version in my laptop is Windows 10 Home but not Professional. I think Sandbox doesn't work here.

**Comment by taooceros** [comment]
![image](https://user-images.githubusercontent.com/45326534/90721743-01e27900-e2ec-11ea-8f8b-4760d5bb26b5.png)
I think the folders in other folders except the quick access folder may be fine.

**Comment by jjw24** [comment]
All good, I will boot up a VM to investigate. 

Just for some more confirmation as well:
- Do these errored results work when you select them?

- from your screenshot it looks like program images from the Program plugin are working yeah?

**Comment by taooceros** [comment]
> All good, I will boot up a VM to investigate.
> 
> Just for some more confirmation as well:
> 
> * Do these errored results work when you select them?.
> * from your screenshot it looks like program images from the Program plugin are working yeah?

I have answered these questions.
> For those folder and files without image, I enter the selection and It results only the path, and nothing else like this. But for those UWP program loading image fail, I can open them with Flow.

The program plugin works well except for a few UWP programs' images loading.

**Comment by jjw24** [comment]
Hey @taooceros a fix is put in. If you like you can test it out for me as well, download the zip version from [here] (https://ci.appveyor.com/api/buildjobs/dbn62qcfixrnv5w3/artifacts/Output%2FPackages%2FFlow-Launcher-v1.2.0.zip) (you will need to install the .Net Core 3.1 runtime, see readme for this build)

**Comment by taooceros** [comment]
> Hey @taooceros a fix is put in. If you like you can test it out for me as well, download the zip version from [here](https://ci.appveyor.com/api/buildjobs/dbn62qcfixrnv5w3/artifacts/Output%2FPackages%2FFlow-Launcher-v1.2.0.zip) (you will need to install the .Net Core 3.1 runtime for this build, see readme)

Thank you! I have installed .Net 5 preview in my computer, and I think that should work as well. I will reach out back after I have tested it.

**Comment by taooceros** [comment]
Hey, this fix works well! Nice Job!

Yet, I have found a new issue as a benefit of this fix. It seems that the explorer plugin doesn't support special characters (I am not sure whether it is because the special character is on the end).
![image](https://user-images.githubusercontent.com/45326534/90998343-7c660e00-e5f6-11ea-8341-dffba9f984ae.png)

The path displays under the selection has removed the latest special character.

**Comment by jjw24** [comment]
> Hey, this fix works well! Nice Job!
> 
> Yet, I have found a new issue as a benefit of this fix. It seems that the explorer plugin doesn't support special characters (I am not sure whether it is because the special character is on the end).
> ![image](https://user-images.githubusercontent.com/45326534/90998343-7c660e00-e5f6-11ea-8341-dffba9f984ae.png)
> 
> The path displays under the selection has removed the latest special character.

Good find. Could you put this new one in a separate issue and I can close this current one once fix is approved and merged.

**Comment by taooceros** [comment]
ok!
